s3-import-commons/src/test/java/no/unit/nva/publication/s3imports/FilenameEventEmitterTest.java

```diff
@@ -61,19 +65,22 @@
     private static final Context CONTEXT = mock(Context.class);
     public static final String LIST_ALL_FILES = ".";
     public static final URI EMPTY_URI = null;
+    public static final Instant FIXED_INSTANT = Instant.parse("2017-02-03T11:25:30.00Z");
```

Comment:
It is a bit worrying that you use the same string everywhere. It could be that we are reading the value from somewhere else. You can make the fixedInstant into a field that gets updated (Instant.now()) before each test to ensure some randomness